### ISSUE TITLE
why size is octetIndex In the Encode function for Q922_Frame

### ISSUE THREAD

**Issue by ms347397173** [willamowius/h323plus#34]
rather than octetIndex +1 ？
![image](https://github.com/willamowius/h323plus/assets/15975831/7e34b366-50ba-43af-aef6-1730fa9a1d63)

**Comment by willamowius** [comment]
Thanks for reporting this. I don't know anybody who used this part of the code in the past 15+ years.
Did you actually use Q.922 and tested with size = octetIndex and size = octetIndex +1  ? What error did you get and did the +1 fix it ?

**Comment by ms347397173** [comment]
octetIndex is the index that has been written，size as buffer length needs to be +1 rather than equal，
origin example：
![image](https://github.com/willamowius/h323plus/assets/15975831/3114b3a4-3ad0-4155-a91c-28eaec61044b)
because size==19, last octect (buffer[19]) will lack in caller's code.

+1 example：
![image](https://github.com/willamowius/h323plus/assets/15975831/001618a1-38ee-4e61-879c-20790f245e5e)
this is correct.

pls  confirm this code,thank you.

**Comment by willamowius** [comment]
Let me rephrase my question: Did you actually try the code change ?
A small piece of code that encodes and decodes a frame that shows the error ?
